lib/bolt/config/options.rb

```diff
@@ -150,7 +150,8 @@ module Options
               description: "Load scripts from the `scripts/` directory of a module.",
               type: [TrueClass, FalseClass],
               _example: true,
-              _default: false
+              _default: false,
+              _deprecation: true
```

Comment:
In thinking more about this, I think the messaging for this is a little tricky. "Deprecation" makes it sound like it's *still* supported, but won't be in the future. I think we could either:
1. Just leave the deprecation warning, it's close enough to the truth
2. Deprecate the option and also raise a warning explaining that it's not honored anymore. This is nice because it'd still be documented.
3. Remove the option altogether. That'll raise a warning that `file_paths` isn't recognized. We could either:
  a. Raise an additional warning explaining it isn't honored anymore
  b. My preference, raise a custom warning that `file_paths` will be ignored, and Bolt will behave as if it's true.

I don't feel too strongly about which  we do, but just "deprecating" the option doesn't feel 100% right.